File: 프로그래머스/Lv.1/약수의 개수와 덧셈/약수의 개수와 덧셈.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int left, int right)
{
  int answer = 0;
  int result = 0;
  for (int i = left; i <= right; i++)
  {
    for (int j = 1; j <= i; j++)
    {
      if (i % j == 0)
      {
        result++;
      }
    }
    if (result % 2 == 0)
    {
      answer = answer + i;
    }
    else
    {
      answer = answer - i;
    }
    result = 0;
  }
  return answer;
}

File: 프로그래머스/Lv.1/약수의 개수와 덧셈/약수의 개수와 덧셈.js
function solution(left, right) {
  let answer = 0;
  let result = 0;
  for (let i = left; i <= right; i++) {
    for (let j = 1; j <= i; j++) {
      if (i % j === 0) {
        result++;
      }
    }
    if (result % 2 === 0) {
      answer += i;
    } else {
      answer -= i;
    }
    result = 0;
  }
  return answer;
}
